doc/architecture/decisions/0004-reference-data-sync.md

```diff
@@ -270,8 +270,20 @@ Authorised requests do not require any roles.
 ]
 ```
 
+#### Get details of an incentive level globally
+NB: Inactive levels are also returned
```

Comment:
I think this is sensible as the [PrisonAPI endpoint we call](https://github.com/ministryofjustice/hmpps-incentives-api/blob/66f95d189c6416a9c1de7d1d331e99612586f0bb/src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonApiService.kt#L57) returns active & inactive levels

Reply:
Perhaps I was going too far with tidying up the external interface :)